addons/binding/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/internal/utils/HttpUtils.java

```diff
@@ -33,7 +35,7 @@
 public class HttpUtils {
     private static Logger logger = LoggerFactory.getLogger(HttpUtils.class);
 
-    private static int TIMEOUT = 5000;
+    private static int timeout = 5000;
```

Comment:
A little background, maybe you are aware, the existing thing was almost a constant in java. The only thing to complete it was adding a `final` modifier. You could consider adding the missing `final` to the class level (almost) constants and keep them uppercase.